tests/log_util_test.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import inspect
+import unittest
+
+from perfkitbenchmarker import log_util
+from perfkitbenchmarker import vm_util
+
+
+class LogUtilTestCase(unittest.TestCase):
+  """Tests exercising the utilities in log_util.
+  """
```

Comment:
Recommend adding a test for passing an empty string to `LabelExtension`, as it's handled differently.